.stylelintrc.json

```diff
@@ -1,20 +0,0 @@
-{
-  "extends": ["stylelint-config-standard"],
-  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
-  "rules": {
-    "at-rule-no-unknown": [
-      true,
-      {
-        "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
-      }
-    ],
-    "scss/at-rule-no-unknown":  [
-      true,
-      {
-        "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
-      }
-    ],
-    "csstree/validator": true
-  },
-  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
-}
```

Comment:
- [ ] Hello @Habtamu210 the reason is you removed configuration of .stylelintrc.json, Kindly add it again https://github.com/microverseinc/linters-config/blob/master/javascript/.stylelintrc.json